README.md

```diff
@@ -25,77 +35,84 @@ Add this dependency to your project's POM:
  
-
 #### Standard Integration
-If you're using [standard (client and server) integration](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/guides/publisher-client-side.md):
+
+If you're using [standard (client and server) integration](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/guides/publisher-client-side.md), follow these steps. **(gwh/JN_03 definitely only one step?)**
+
 1. Send this identity as a JSON string back to the client (to use in the [identity field](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/sdks/client-side-identity.md#initopts-object-void)) using: `identity.getJsonString()`
 
 #### Server-Only Integration
+
 If you're using [server-only integration](https://github.com/UnifiedID2/uid2docs/blob/main/api/v2/guides/custom-publisher-integration.md):
 
 1. Store this identity as a JSON string in the user's session, using: `identity.getJsonString()`
 2. To use the user's UID2 token, use `identity.getAdvertisingToken()`
 
 3. When the user accesses another page, or on a timer, determine whether a refresh is needed:
-   1. Retrieve the identity JSON string from the user's session, then call:
+   1. Retrieve the identity JSON string from the user's session, then call: **(gwh/JN_04 should this be two steps?)**
 
       `IdentityTokens identity = IdentityTokens.fromJsonString(identityJsonString);`
-   2. Determine if the identity is refreshable (ie, the refresh token hasn't expired):
+   2. Determine if the identity is refreshable (that is, the refresh token hasn't expired):
 
-      ` if (identity == null || !identity.isRefreshable()) { we must no longer use this identity (eg, remove this identity from the user's session) }`
+      ` if (identity == null || !identity.isRefreshable()) { we must no longer use this identity (for example, remove this identity from the user's session) }`
    3. Determine if a refresh is needed:
 
       `if (identity.isDueForRefresh()) {..}`
-4. If a refresh is needed, call:
+4. If a refresh is needed, call: **(gwh/JN_05 can we state what this is doing? Initiate token refresh?)**
  
    `TokenRefreshResponse tokenRefreshResponse = publisherUid2Client.refreshToken(identity);`
  
 5. You should then store `tokenRefreshResponse.getIdentityJsonString()` in the user's session. If the user has opted out, this method will return null, indicating that the user's identity should be removed from their session. (Optout can be confirmed via `tokenRefreshResponse.isOptout()`.)
 
 
-### Advanced usage
+### Advanced Usage
 
-1. Create an instance of PublisherUid2Helper as an instance variable. 
+1. Create an instance of PublisherUid2Helper as an instance variable:
 
     `private final PublisherUid2Helper publisherUid2Helper = new PublisherUid2Helper(UID2_SECRET_KEY);`
-2. When a user authenticates and authorizes the creation of a UID2:
+2. When a user authenticates and authorizes the creation of a UID2: **(gwh/JN_06 can we state what this is doing?)**
```

Comment:
This is step 1-b and 1-c here: https://github.com/IABTechLab/uid2docs/blob/main/api/v2/guides/custom-publisher-integration.md

Reply:
Trying again... let's discuss.